eng/scripts/automation/Invoke-MockTest.ps1

```diff
@@ -0,0 +1,50 @@
+#Requires -Version 7.0
+param(
+    [string]$inputJsonFile,
+    [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot .. MgmtTestLib.ps1)
+Write-output "inputfile:$inputJsonFile, outfile:$outputJsonFile"
+$inputstring = Get-Content $inputJsonFile | Out-String
+Write-Output "input parameter: $inputstring"
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$packageFolder = $inputJson.packageFolder
+$packageFolder = $packageFolder -replace "\\", "/"
+
+$runLocalMockServer = $false
+if ([string]::IsNullOrEmpty($inputJson.mockServerHost)) {
+    $runLocalMockServer = $true
+}
+Write-Host "##[command]Generate example and Mock Test " $packageFolder
+Set-Location $packageFolder
+Invoke-MgmtTestgen -sdkDirectory $packageFolder -autorestPath $packageFolder/autorest.md -generateExample -generateMockTest -tidy
```

Comment:
`generateExample` is no need as it will be generated during the code generation period. Also I think `-format -vet` is needed for the compiling check.